app/policies/page_policy.rb

```diff
@@ -8,32 +8,26 @@ def show?
     user_is_member?
   end
 
-
```

Comment:
same comment re: formatting and style:  please don't delete the blank lines between methods in this file